docs: fixing syntax (#162)
nervosnetwork/light-godwoken: docs/advanced-settings.md

How: 

1. Go to Godwoken Bridge: https://light-godwoken-git-advanced-settings-cryptape.vercel.app/#/v1
2. Open developer tools in browser and navigate to `Application`, find `local storage` in left panel: 
<img width="1914" alt="image" src="https://user-images.githubusercontent.com/3870972/172973026-0b0a4900-5b73-4483-8b95-b516a25b4f6c.png">

> 3. Change `MIN_CANCEL_DEPOSIT_TIME` to an acceptable time (I change it to `1`) to wait for the deposit to expire. The time unit is in seconds. Then you can **refresh** the page and fire a deposit. After a short time, the deposit should be canceled.

Reviewer: ```suggestion
3. Change `MIN_CANCEL_DEPOSIT_TIME` to an acceptable time (I change it to `1`) to wait for the deposit to expire. The time unit is in seconds. Then you can **refresh** the page and fire a deposit. After a short time, the deposit will become cancelable. 
```

when a deposit time exceeds `MIN_CANCEL_DEPOSIT_TIME`, the deposit will become cancelable, and the canceling is triggered by the user
